docs/guides/universal-profile/01-create-profile.md

```diff
@@ -182,13 +183,14 @@ const web3 = new Web3();
 
 const PRIVATE_KEY = '0x...'; // your EOA private key (previously created)
 const myEOA = web3.eth.accounts.privateKeyToAccount(PRIVATE_KEY);
-// {
-//     address: "0x...",
-//     privateKey: "0x...",
-//     signTransaction: function(tx){...},
-//     sign: function(data){...},
-//     encrypt: function(password){...}
-// }
+
+> {
+    address: "0x...",
+    privateKey: "0x...",
+    signTransaction: function(tx){...},
+    sign: function(data){...},
+    encrypt: function(password){...}
+}
```

Comment:
```suggestion
/**
{
    address: "0x...",
    privateKey: "0x...",
    signTransaction: function(tx){...},
    sign: function(data){...},
    encrypt: function(password){...}
}
*/
```

Reply:
This change was requested by @frozeman, to put more emphasis on the outputs.